app/models/health/agency_performance.rb

```diff
@@ -30,6 +32,12 @@ def agency_counts
         with_signed_careplans = careplan_dates.select{ |p_id, _| p_id.in?(patient_ids) }.keys
         without_signed_careplans = patient_ids - with_signed_careplans
 
+        with_careplans_in_122_days = patient_ids.select do |p_id|
+          careplan_date = careplan_dates[p_id]&.to_date
+          enrollment_date = patient_referrals[p_id][1]
+          careplan_date.present? && (careplan_date - enrollment_date).to_i <= 122
```

Comment:
@awisema, per our recent call with BHCHP, `careplan_date` should be the date of the QA that represents the signature.